UI.cs

```diff
@@ -50,6 +50,7 @@ public static bool ValidTiers(IReadOnlyCollection<int> tiers) =>
     
     public static void SetUpLevelInput()
     {
+        var slider = levelselect.AddSlider(new Info("Slider"), 0, 0, 5, 0.1f, new Vector2(100, 100));
```

Comment:
what is this for

Reply:
no clue tbh, it was already there, idk why it shows as new